### DISCUSSION TITLE
Vi er videre nu kører vi 1.32 .... Nogen der ved hvorfor alle versioner af Wordmat 1.31.2 optræder som Virus!

### DISCUSSION THREAD

**Discussion by IBG-IT** [Eduap-com/WordMat#247] [Q&A]
Alle versioner som downloades optræder som virus både på MAC og PC og hos forskellige anti-virus producenter...

Nogen der har en ide om "det er virus" eller hvordan det nemmest omgås så Wordmat ved installation ikke optræder som en virus?

**Comment by IBG-IT** [comment]
Nogen der har fået en ren version, hvor Wordmat 1.31.3 (25/10.24)  ikke har en virusdel i sig ?

Vi har kigget på nettet og spurgt forskellige "kloge"  de mener der er virus i Wordmat 1.31.3 (og denne virus fanges på både domainmaskiner og ikke domain maskiner som kører Windows og som bruger anti-virus produktet "Microsoft Defender"

Virusen hedder:
'Wacatac' malware was prevented was detected by Microsoft Defender for Endpoint
og ligger i Wordmat skabelonen og kaldes via et kald Wordmat laver!
trojan:Script/Wacatac.H!ml   og ligger i wordmat.dotm

Det tyder på der ikke er virus i version 1.30 - den fanger ingen "virus"

**Reply by JalteJKR** [reply]
Kan du uddybe “spurgt forskellige kloge”, hvem præcis? Og hvad baserer de deres konklusion om, at der er en virus i denne WordMat-version, på? Jeg spørger ikke, fordi jeg har en holdning til, om det er rigtigt eller forkert, men bare fordi jeg selv er nysgerrig, om det mon kan være sandt (at det ikke er en falsk positiv), og samtidig forholder mig lidt kritisk til det, du skriver.

**Reply by UffeJakobsen** [reply]
Jeg synes at det er meget påfaldende at man ikke kan downloade nogen af 1.31.x udgaverene mere - specielt når man stadig kan downloade ældre versioner som 1.30, 1.29 osv... Det kunne man jo godt tolke som et forsøg på at skjule problemer med 1.31.x

**Comment by UffeJakobsen** [comment]
Jeg ser samme problem...

Jeg kører version 1.31.3 - og opdager at 1.31.3 nu er fjernet fra download siden... ingen forklaring
Nyeste er nu 1.31.2... men jvf andre issues så er der også problemer med den 1.31.2 og 1.31

Jeg har stillet spørgsmål i issue #265

**Reply by soegaard** [reply]
Hej Uffe,

Du kan hente 1.31.3 her:
https://github.com/Eduap-com/WordMat/releases

Analysen fra en tilfælde online virus checker (VirusTotal - som jeg i øvrigt ikke kender):

<img width="1378" alt="image" src="https://github.com/user-attachments/assets/b2e5adbd-d0d4-4e35-af47-42c3a6cac6b4" />

Det ser ret uskyldigt ud.

Der er to røde i forbindelse med "Office Start folder", men det er nu ret naturligt for en Word-udvidelse.

Det er heller ikke underligt, at der bruges Office-makroer i WordMat ej heller er det underligt at der kopieres filer
(for at gøre installationen af de medfølgende programmer nem).

Mvh,
Jens Axel

**Reply by UffeJakobsen** [reply]
Jeg bruger kun de releases som ligger på den officielle EDUAP download side: https://www.eduap.com/downloadwordmat/ og her er ingen 1.31.x releases mere - det var der tidligere... Jeg har flere gange prøvet at spørge til hvorfor de er fjernet - der bliver svaret med stilhed.

**Reply by soegaard** [reply]
> Jeg bruger kun de releases som ligger på den officielle EDUAP download side:
Det er de samme. Prøv at kopiere linket fra den officielle EDUAP download-side
og sæt det ind i adressefeltet. Du vil se, at filen er hosted på Github.

**Reply by UffeJakobsen** [reply]
Hvor det hostes er sådan set irrelevant i forhold til det jeg spørger om. Hvis du scroller hele vejen ned af den side jeg har henvist til så vil ud -udover version 1.32 i toppen - se en masse af de gamle releases som er ældre end 1.31.x - hvorfor har man fjernet 1.31.x derfra ?

**Reply by soegaard** [reply]
> Hvor det hostes er sådan set irrelevant i forhold til det jeg spørger om.
Ja - men det er der ikke andre end Mikael, der kan svare på.

Men da du er på jagt efter den originale installationsfil er der ikke helt ved siden af at henvise til siden med releases.

Vær opmærksom på, at du skal folde "Assets" ud.

<img width="1001" alt="image" src="https://github.com/user-attachments/assets/81452738-6ae0-4fde-81a4-495ec5f0058a" />

**Reply by UffeJakobsen** [reply]
Du misforstår - Jeg er ikke på jagt efter den.
Jeg downloadede 1.31.x versionerne dengang de lå på den officielle EDUAP side - og var de nyeste releases. 
Nu er de fjernet fra den side - og man ønsker åbentbart ikke at forklare hvorfor lige netop de releases er fjernet. Når siden nu indeholder et helt arkiv af gamle releases - hvorfor så ikke 1.31.x ?
Man kunne jo få den tanke at det faktisk var fordi at "man" ved en fejl har distribueret en virus sammen med 1.31.x udgaverne. 
Der er masse virus checkers der reagerer på lige netop 1.31.x udgaverne - inklusive Microsoft Defender, TrendMicro osv  - og håndteringen (stilhed) øger måske ikke ligefrem  tilliden.

**Comment by soegaard** [comment]
> Jeg downloadede 1.31.x versionerne dengang de lå på den officielle EDUAP side - og var de nyeste releases.
Nu er de fjernet fra den side - og man ønsker åbentbart ikke at forklare hvorfor lige netop de releases er fjernet. Når siden nu indeholder et helt arkiv af gamle releases - hvorfor så ikke 1.31.x ?

Ingen anelse - men hvis du har filen, så hvorfor så ikke afprøve din teori ved at virusscanne filen?
Vis gerne resultaterne her.

> Der er masse virus checkers der reagerer på lige netop 1.31.x udgaverne - inklusive Microsoft Defender, TrendMicro osv - og håndteringen (stilhed) øger måske ikke ligefrem tilliden.

Hvordan reageres der? 

Er det i stil med resultaterne fra VirusTotal (så virker det harmløst).

Eller er det en konkret fil der indeholder virus (så er det straks værre).

**Reply by UffeJakobsen** [reply]
De forskellige virus scannere fjerner dele af Wordmat installationen (karantæne) - så den ikke længere virker - det er sket gentagende gange med 1.31.x
Jeg har sådan set ikke lyst til at eksperimentere (mere) med en maskine der gerne skal virke hver dag i skolen.

**Comment by soegaard** [comment]
Jeg forsøgte at finde et par online checkere (har ikke selv Windows). 
Her er resultaterne fra 1.31.2.

FSecure:
<img width="1084" alt="image" src="https://github.com/user-attachments/assets/ffa1e181-e5d4-43dd-b106-7ad3a3b9f6d0" />

Jotti (som tjekker med flere scannere):
<img width="1178" alt="image" src="https://github.com/user-attachments/assets/9e826f7f-f596-40f5-bce1-49784922c012" />

**Comment by IBG-IT** [comment]
Vi er videre da vi nu kører 1.32 overalt.
